Compile bug: Unable to build

### Git commit

https://github.com/ggml-org/llama.cpp/releases/tag/b6344



### Operating systems

Linux

### GGML backends

HIP, Vulkan

### Problem description & steps to reproduce

Unable to build starting from b6344

### First Bad Commit

b6344 start, cannot build .

### Compile command

```shell
INSTALL_DIR_PREFIX_SDK_ROOT="/opt/rocm_sdk_612"

HIPCXX="$(hipconfig -l)/clang"

HIP_PATH="$(hipconfig -R)"

BLAS_INCLUDE_DIRS="/opt/rocm_sdk_612/include/openblas"

cmake -S . -B build \
    -DBLAS_INCLUDE_DIRS=${BLAS_INCLUDE_DIRS} \
    -DCMAKE_INSTALL_PREFIX=${INSTALL_DIR_PREFIX_SDK_ROOT} \
    -DROCM_PATH=${INSTALL_DIR_PREFIX_SDK_ROOT} \
    -DGGML_CUDA_FA_ALL_QUANTS=ON \
    -DGGML_CCACHE=OFF \
    -DGGML_VULKAN=ON \
    -DGGML_HIP=ON \
    -DGGML_FLASH_ATTN=ON \
    -DGGML_HIPBLAS=ON \
    -DGGML_USE_ROCM_WMMA=ON \
    -DGGML_HIP_ROCWMMA_FATTN=ON \
    -DGGML_CUDA_FA_ALL_QUANTS=ON \
    -DLLAMA_SERVER_SSL=ON \
    -DAMDGPU_TARGETS=gfx1151 \
    -DCMAKE_BUILD_TYPE=Release \
    -DCMAKE_CXX_FLAGS="-O3" \
    -DCMAKE_C_FLAGS="-O3" \
    -DLLAMA_CUDA_ENABLE_UNIFIED_MEMORY=ON \
    -DGGML_CURL=ON \
    -DLLAMA_CURL=ON \
    -DGGML_BLAS=ON \
    -DGGML_BLAS_VENDOR=OpenBLAS \
    -DGGML_HIP_UMA=ON \
    -DGGML_CUDA_ENABLE_UNIFIED_MEMORY=ON \
    -DLLAMA_CUDA_ENABLE_UNIFIED_MEMORY=ON \
    -DCMAKE_PREFIX_PATH="${INSTALL_DIR_PREFIX_SDK_ROOT}/lib64/cmake:${INSTALL_DIR_PREFIX_SDK_ROOT}/lib/cmake"
```

### Relevant log output

```log
[  1%] Building C object examples/gguf-hash/CMakeFiles/xxhash.dir/deps/xxhash/xxhash.c.o
[  1%] Building C object examples/gguf-hash/CMakeFiles/sha256.dir/deps/sha256/sha256.c.o
[  2%] Building C object ggml/src/CMakeFiles/ggml-base.dir/ggml.c.o
[  2%] Building CXX object ggml/src/CMakeFiles/ggml-base.dir/ggml-backend.cpp.o
[  2%] Building C object ggml/src/CMakeFiles/ggml-base.dir/ggml-alloc.c.o
[  2%] Building C object examples/gguf-hash/CMakeFiles/sha1.dir/deps/sha1/sha1.c.o
[  2%] Building CXX object ggml/src/CMakeFiles/ggml-base.dir/ggml-threading.cpp.o
[  2%] Building CXX object common/CMakeFiles/build_info.dir/build-info.cpp.o
[  2%] Building CXX object ggml/src/CMakeFiles/ggml-base.dir/ggml.cpp.o
[  2%] Building C object ggml/src/CMakeFiles/ggml-base.dir/ggml-quants.c.o
[  2%] Building CXX object tools/mtmd/CMakeFiles/llama-llava-cli.dir/deprecation-warning.cpp.o
[  3%] Building CXX object ggml/src/CMakeFiles/ggml-base.dir/ggml-opt.cpp.o
[  4%] Building CXX object tools/mtmd/CMakeFiles/llama-minicpmv-cli.dir/deprecation-warning.cpp.o
[  4%] Building CXX object tools/mtmd/CMakeFiles/llama-qwen2vl-cli.dir/deprecation-warning.cpp.o
[  4%] Creating directories for 'vulkan-shaders-gen'
[  4%] Building CXX object tools/mtmd/CMakeFiles/llama-gemma3-cli.dir/deprecation-warning.cpp.o
[  4%] Building CXX object ggml/src/CMakeFiles/ggml-base.dir/gguf.cpp.o
[  4%] Built target sha256
[  4%] Built target sha1
[  4%] Built target build_info
[  4%] Built target xxhash
[  4%] Linking CXX executable ../../bin/llama-qwen2vl-cli
[  4%] Linking CXX executable ../../bin/llama-llava-cli
[  5%] Linking CXX executable ../../bin/llama-minicpmv-cli
[  5%] Linking CXX executable ../../bin/llama-gemma3-cli
[  5%] No download step for 'vulkan-shaders-gen'
[  5%] Linking CXX shared library ../../bin/libggml-base.so
[  5%] No update step for 'vulkan-shaders-gen'
[  5%] No patch step for 'vulkan-shaders-gen'
[  5%] Built target llama-llava-cli
[  5%] Built target llama-qwen2vl-cli
[  5%] Built target llama-minicpmv-cli
[  5%] Built target llama-gemma3-cli
[  5%] Built target ggml-base
[  5%] Performing configure step for 'vulkan-shaders-gen'
[  5%] Building CXX object ggml/src/CMakeFiles/ggml-cpu.dir/ggml-cpu/binary-ops.cpp.o
[  5%] Building CXX object ggml/src/CMakeFiles/ggml-cpu.dir/ggml-cpu/traits.cpp.o
[  5%] Building C object ggml/src/CMakeFiles/ggml-cpu.dir/ggml-cpu/quants.c.o
[  5%] Building CXX object ggml/src/CMakeFiles/ggml-cpu.dir/ggml-cpu/llamafile/sgemm.cpp.o
[  5%] Building CXX object ggml/src/CMakeFiles/ggml-cpu.dir/ggml-cpu/vec.cpp.o
[  6%] Building CXX object ggml/src/ggml-blas/CMakeFiles/ggml-blas.dir/ggml-blas.cpp.o
[  6%] Building C object ggml/src/CMakeFiles/ggml-cpu.dir/ggml-cpu/ggml-cpu.c.o
[  6%] Building CXX object ggml/src/CMakeFiles/ggml-cpu.dir/ggml-cpu/ggml-cpu.cpp.o
[  7%] Building CXX object ggml/src/CMakeFiles/ggml-cpu.dir/ggml-cpu/repack.cpp.o
[  7%] Building CXX object ggml/src/CMakeFiles/ggml-cpu.dir/ggml-cpu/amx/amx.cpp.o
[  8%] Building CXX object ggml/src/CMakeFiles/ggml-cpu.dir/ggml-cpu/amx/mmq.cpp.o
[  8%] Building CXX object ggml/src/CMakeFiles/ggml-cpu.dir/ggml-cpu/unary-ops.cpp.o
[  8%] Building CXX object ggml/src/CMakeFiles/ggml-cpu.dir/ggml-cpu/hbm.cpp.o
[  9%] Building CXX object ggml/src/CMakeFiles/ggml-cpu.dir/ggml-cpu/ops.cpp.o
[  9%] Building C object ggml/src/CMakeFiles/ggml-cpu.dir/ggml-cpu/arch/x86/quants.c.o
[  9%] Building CXX object ggml/src/CMakeFiles/ggml-cpu.dir/ggml-cpu/arch/x86/repack.cpp.o
[  9%] Linking CXX shared library ../../../bin/libggml-blas.so
[ 10%] Linking CXX shared library ../../bin/libggml-cpu.so
-- The C compiler identification is GNU 14.2.1
[ 10%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/argmax.cu.o
[ 10%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/binbcast.cu.o
[ 11%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/argsort.cu.o
[ 11%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/concat.cu.o
[ 12%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/conv-transpose-1d.cu.o
[ 12%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/add-id.cu.o
[ 12%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/acc.cu.o
[ 12%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/arange.cu.o
[ 12%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/clamp.cu.o
[ 12%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/conv2d-dw.cu.o
[ 12%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/conv2d-transpose.cu.o
[ 12%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/convert.cu.o
[ 12%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/conv2d.cu.o
[ 12%] Built target ggml-blas
[ 12%] Built target ggml-cpu
[ 13%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/count-equal.cu.o
[ 13%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/cpy.cu.o
-- The CXX compiler identification is GNU 14.2.1
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /usr/lib64/ccache/cc - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /usr/lib64/ccache/c++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD - Success
-- Found Threads: TRUE
-- Enabling coopmat glslc support
-- Configuring done (0.3s)
-- Generating done (0.0s)
-- Build files have been written to: /home/xuegao/app/ai/llama.cpp/build/ggml/src/ggml-vulkan/vulkan-shaders-gen-prefix/src/vulkan-shaders-gen-build
[ 13%] Performing build step for 'vulkan-shaders-gen'
[ 50%] Building CXX object CMakeFiles/vulkan-shaders-gen.dir/vulkan-shaders-gen.cpp.o
[100%] Linking CXX executable vulkan-shaders-gen
[100%] Built target vulkan-shaders-gen
[ 14%] Performing install step for 'vulkan-shaders-gen'
-- Installing: /home/xuegao/app/ai/llama.cpp/build/Release/./vulkan-shaders-gen
[ 15%] Completed 'vulkan-shaders-gen'
[ 15%] Built target vulkan-shaders-gen
[ 15%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/cross-entropy-loss.cu.o
[ 15%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/diagmask.cu.o
[ 16%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/fattn-tile-f16.cu.o
[ 16%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/fattn-tile-f32.cu.o
[ 16%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/fattn-wmma-f16.cu.o
[ 16%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/fattn.cu.o
[ 17%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/getrows.cu.o
[ 17%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/ggml-cuda.cu.o
[ 17%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/gla.cu.o
[ 17%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/im2col.cu.o
[ 17%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/mean.cu.o
[ 18%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/mmf.cu.o
[ 18%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/mmq.cu.o
[ 18%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/mmvf.cu.o
[ 18%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/mmvq.cu.o
[ 19%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/norm.cu.o
[ 19%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/opt-step-adamw.cu.o
[ 19%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/opt-step-sgd.cu.o
[ 19%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/out-prod.cu.o
[ 19%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/pad.cu.o
[ 20%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/pad_reflect_1d.cu.o
[ 20%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/pool2d.cu.o
[ 20%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/quantize.cu.o
[ 20%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/roll.cu.o
[ 21%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/rope.cu.o
[ 21%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/scale.cu.o
[ 21%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/set-rows.cu.o
[ 21%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/softcap.cu.o
[ 22%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/softmax.cu.o
[ 22%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/ssm-conv.cu.o
[ 22%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/ssm-scan.cu.o
[ 22%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/sum.cu.o
[ 22%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/sumrows.cu.o
[ 23%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/tsembd.cu.o
[ 23%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/unary.cu.o
[ 23%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/upscale.cu.o
[ 23%] Generate vulkan shaders
ggml_vulkan: Generating and compiling shaders to SPIR-V
[ 23%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/wkv.cu.o
[ 24%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/template-instances/fattn-mma-f16-instance-ncols1_1-ncols2_16.cu.o
[ 24%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/template-instances/fattn-mma-f16-instance-ncols1_1-ncols2_8.cu.o
[ 24%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/template-instances/fattn-mma-f16-instance-ncols1_16-ncols2_1.cu.o
[ 24%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/template-instances/fattn-mma-f16-instance-ncols1_16-ncols2_2.cu.o
[ 24%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/template-instances/fattn-mma-f16-instance-ncols1_16-ncols2_4.cu.o
[ 25%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/template-instances/fattn-mma-f16-instance-ncols1_2-ncols2_16.cu.o
[ 25%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/template-instances/fattn-mma-f16-instance-ncols1_2-ncols2_4.cu.o
[ 25%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/template-instances/fattn-mma-f16-instance-ncols1_2-ncols2_8.cu.o
[ 25%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/template-instances/fattn-mma-f16-instance-ncols1_32-ncols2_1.cu.o
[ 26%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/template-instances/fattn-mma-f16-instance-ncols1_32-ncols2_2.cu.o
[ 26%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/template-instances/fattn-mma-f16-instance-ncols1_4-ncols2_16.cu.o
[ 26%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/template-instances/fattn-mma-f16-instance-ncols1_4-ncols2_2.cu.o
[ 26%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/template-instances/fattn-mma-f16-instance-ncols1_4-ncols2_4.cu.o
[ 27%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/template-instances/fattn-mma-f16-instance-ncols1_4-ncols2_8.cu.o
[ 27%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/template-instances/fattn-mma-f16-instance-ncols1_8-ncols2_1.cu.o
[ 27%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/template-instances/fattn-mma-f16-instance-ncols1_64-ncols2_1.cu.o
[ 27%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/template-instances/fattn-mma-f16-instance-ncols1_8-ncols2_2.cu.o
[ 27%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/template-instances/fattn-mma-f16-instance-ncols1_8-ncols2_4.cu.o
[ 28%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/template-instances/fattn-mma-f16-instance-ncols1_8-ncols2_8.cu.o
[ 28%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/template-instances/mmq-instance-iq1_s.cu.o
[ 28%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/template-instances/mmq-instance-iq2_s.cu.o
[ 28%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/template-instances/mmq-instance-iq2_xs.cu.o
[ 29%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/template-instances/mmq-instance-iq2_xxs.cu.o
[ 29%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/template-instances/mmq-instance-iq3_s.cu.o
[ 29%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/template-instances/mmq-instance-iq3_xxs.cu.o
[ 29%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/template-instances/mmq-instance-iq4_nl.cu.o
[ 29%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/template-instances/mmq-instance-iq4_xs.cu.o
[ 30%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/template-instances/mmq-instance-mxfp4.cu.o
[ 30%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/template-instances/mmq-instance-q2_k.cu.o
[ 30%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/template-instances/mmq-instance-q3_k.cu.o
[ 30%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/template-instances/mmq-instance-q4_0.cu.o
[ 31%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/template-instances/mmq-instance-q4_1.cu.o
[ 31%] Building CXX object ggml/src/ggml-vulkan/CMakeFiles/ggml-vulkan.dir/ggml-vulkan.cpp.o
/home/xuegao/app/ai/llama.cpp/ggml/src/ggml-vulkan/ggml-vulkan.cpp: In function ‘void ggml_vk_load_shaders(vk_device&)’:
/home/xuegao/app/ai/llama.cpp/ggml/src/ggml-vulkan/ggml-vulkan.cpp:2930:69: 警告：建议在‘||’的操作数中出现的‘&&’前后加上括号 [-Wparentheses]
 2930 |                                     (!device->subgroup_size_control && device->subgroup_size >= 16 ||
      |                                      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
[ 31%] Building CXX object ggml/src/ggml-vulkan/CMakeFiles/ggml-vulkan.dir/ggml-vulkan-shaders.cpp.o
[ 31%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/template-instances/mmq-instance-q4_k.cu.o
[ 31%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/template-instances/mmq-instance-q5_0.cu.o
[ 31%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/template-instances/mmq-instance-q5_1.cu.o
[ 32%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/template-instances/mmq-instance-q5_k.cu.o
[ 32%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/template-instances/mmq-instance-q6_k.cu.o
[ 32%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/template-instances/mmq-instance-q8_0.cu.o
[ 32%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/template-instances/fattn-vec-f16-instance-hs128-f16-f16.cu.o
[ 32%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/template-instances/fattn-vec-f16-instance-hs128-f16-q4_0.cu.o
[ 33%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/template-instances/fattn-vec-f16-instance-hs128-f16-q4_1.cu.o
[ 33%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/template-instances/fattn-vec-f16-instance-hs128-f16-q5_0.cu.o
[ 33%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/template-instances/fattn-vec-f16-instance-hs128-f16-q5_1.cu.o
[ 33%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/template-instances/fattn-vec-f16-instance-hs128-f16-q8_0.cu.o
[ 34%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/template-instances/fattn-vec-f16-instance-hs128-q4_0-f16.cu.o
[ 34%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/template-instances/fattn-vec-f16-instance-hs128-q4_0-q4_0.cu.o
[ 34%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/template-instances/fattn-vec-f16-instance-hs128-q4_0-q4_1.cu.o
[ 34%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/template-instances/fattn-vec-f16-instance-hs128-q4_0-q5_0.cu.o
[ 34%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/template-instances/fattn-vec-f16-instance-hs128-q4_0-q5_1.cu.o
[ 35%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/template-instances/fattn-vec-f16-instance-hs128-q4_0-q8_0.cu.o
[ 35%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/template-instances/fattn-vec-f16-instance-hs128-q4_1-f16.cu.o
[ 35%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/template-instances/fattn-vec-f16-instance-hs128-q4_1-q4_0.cu.o
[ 35%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/template-instances/fattn-vec-f16-instance-hs128-q4_1-q4_1.cu.o
[ 36%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/template-instances/fattn-vec-f16-instance-hs128-q4_1-q5_0.cu.o
[ 36%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/template-instances/fattn-vec-f16-instance-hs128-q4_1-q5_1.cu.o
[ 36%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/template-instances/fattn-vec-f16-instance-hs128-q4_1-q8_0.cu.o
[ 36%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/template-instances/fattn-vec-f16-instance-hs128-q5_0-f16.cu.o
[ 37%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/template-instances/fattn-vec-f16-instance-hs128-q5_0-q4_0.cu.o
[ 37%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/template-instances/fattn-vec-f16-instance-hs128-q5_0-q4_1.cu.o
/home/xuegao/app/ai/llama.cpp/build/ggml/src/ggml-vulkan/ggml-vulkan-shaders.cpp:2956808: 错误：‘mul_mat_vec_q4_0_q8_1_f32_data’ was not declared in this scope; did you mean ‘mul_mat_vec_q4_0_f32_f32_data’?
2956808 | unsigned char *arr_dmmv_q4_0_q8_1_f32_data[3] = {mul_mat_vec_q4_0_q8_1_f32_data, mul_mat_vec_q4_0_q8_1_f32_subgroup_data, mul_mat_vec_q4_0_q8_1_f32_subgroup_no_shmem_data};
/home/xuegao/app/ai/llama.cpp/build/ggml/src/ggml-vulkan/ggml-vulkan-shaders.cpp:2956808: 错误：‘mul_mat_vec_q4_0_q8_1_f32_subgroup_data’ was not declared in this scope; did you mean ‘mul_mat_vec_q4_0_f32_f32_subgroup_data’?
/home/xuegao/app/ai/llama.cpp/build/ggml/src/ggml-vulkan/ggml-vulkan-shaders.cpp:2956808: 错误：‘mul_mat_vec_q4_0_q8_1_f32_subgroup_no_shmem_data’ was not declared in this scope; did you mean ‘mul_mat_vec_q4_0_f32_f32_subgroup_no_shmem_data’?
/home/xuegao/app/ai/llama.cpp/build/ggml/src/ggml-vulkan/ggml-vulkan-shaders.cpp:2956809: 错误：‘mul_mat_vec_q4_0_q8_1_f32_len’ was not declared in this scope; did you mean ‘mul_mat_vec_q4_0_f32_f32_len’?
2956809 | uint64_t arr_dmmv_q4_0_q8_1_f32_len[3] =  {mul_mat_vec_q4_0_q8_1_f32_len,  mul_mat_vec_q4_0_q8_1_f32_subgroup_len, mul_mat_vec_q4_0_q8_1_f32_subgroup_no_shmem_len};
/home/xuegao/app/ai/llama.cpp/build/ggml/src/ggml-vulkan/ggml-vulkan-shaders.cpp:2956809: 错误：‘mul_mat_vec_q4_0_q8_1_f32_subgroup_len’ was not declared in this scope; did you mean ‘mul_mat_vec_q4_0_f32_f32_subgroup_len’?
/home/xuegao/app/ai/llama.cpp/build/ggml/src/ggml-vulkan/ggml-vulkan-shaders.cpp:2956809: 错误：‘mul_mat_vec_q4_0_q8_1_f32_subgroup_no_shmem_len’ was not declared in this scope; did you mean ‘mul_mat_vec_q4_0_f32_f32_subgroup_no_shmem_len’?
/home/xuegao/app/ai/llama.cpp/build/ggml/src/ggml-vulkan/ggml-vulkan-shaders.cpp:2956810: 错误：‘mul_mat_vec_q4_1_q8_1_f32_data’ was not declared in this scope; did you mean ‘mul_mat_vec_q4_1_f32_f32_data’?
2956810 | unsigned char *arr_dmmv_q4_1_q8_1_f32_data[3] = {mul_mat_vec_q4_1_q8_1_f32_data, mul_mat_vec_q4_1_q8_1_f32_subgroup_data, mul_mat_vec_q4_1_q8_1_f32_subgroup_no_shmem_data};
/home/xuegao/app/ai/llama.cpp/build/ggml/src/ggml-vulkan/ggml-vulkan-shaders.cpp:2956810: 错误：‘mul_mat_vec_q4_1_q8_1_f32_subgroup_data’ was not declared in this scope; did you mean ‘mul_mat_vec_q4_1_f32_f32_subgroup_data’?
/home/xuegao/app/ai/llama.cpp/build/ggml/src/ggml-vulkan/ggml-vulkan-shaders.cpp:2956810: 错误：‘mul_mat_vec_q4_1_q8_1_f32_subgroup_no_shmem_data’ was not declared in this scope; did you mean ‘mul_mat_vec_q4_1_f32_f32_subgroup_no_shmem_data’?
/home/xuegao/app/ai/llama.cpp/build/ggml/src/ggml-vulkan/ggml-vulkan-shaders.cpp:2956811: 错误：‘mul_mat_vec_q4_1_q8_1_f32_len’ was not declared in this scope; did you mean ‘mul_mat_vec_q4_1_f32_f32_len’?
2956811 | uint64_t arr_dmmv_q4_1_q8_1_f32_len[3] =  {mul_mat_vec_q4_1_q8_1_f32_len,  mul_mat_vec_q4_1_q8_1_f32_subgroup_len, mul_mat_vec_q4_1_q8_1_f32_subgroup_no_shmem_len};
/home/xuegao/app/ai/llama.cpp/build/ggml/src/ggml-vulkan/ggml-vulkan-shaders.cpp:2956811: 错误：‘mul_mat_vec_q4_1_q8_1_f32_subgroup_len’ was not declared in this scope; did you mean ‘mul_mat_vec_q4_1_f32_f32_subgroup_len’?
/home/xuegao/app/ai/llama.cpp/build/ggml/src/ggml-vulkan/ggml-vulkan-shaders.cpp:2956811: 错误：‘mul_mat_vec_q4_1_q8_1_f32_subgroup_no_shmem_len’ was not declared in this scope; did you mean ‘mul_mat_vec_q4_1_f32_f32_subgroup_no_shmem_len’?
/home/xuegao/app/ai/llama.cpp/build/ggml/src/ggml-vulkan/ggml-vulkan-shaders.cpp:2956812: 错误：‘mul_mat_vec_q5_0_q8_1_f32_data’ was not declared in this scope; did you mean ‘mul_mat_vec_q5_0_f32_f32_data’?
2956812 | unsigned char *arr_dmmv_q5_0_q8_1_f32_data[3] = {mul_mat_vec_q5_0_q8_1_f32_data, mul_mat_vec_q5_0_q8_1_f32_subgroup_data, mul_mat_vec_q5_0_q8_1_f32_subgroup_no_shmem_data};
/home/xuegao/app/ai/llama.cpp/build/ggml/src/ggml-vulkan/ggml-vulkan-shaders.cpp:2956812: 错误：‘mul_mat_vec_q5_0_q8_1_f32_subgroup_data’ was not declared in this scope; did you mean ‘mul_mat_vec_q5_0_f32_f32_subgroup_data’?
/home/xuegao/app/ai/llama.cpp/build/ggml/src/ggml-vulkan/ggml-vulkan-shaders.cpp:2956812: 错误：‘mul_mat_vec_q5_0_q8_1_f32_subgroup_no_shmem_data’ was not declared in this scope; did you mean ‘mul_mat_vec_q5_0_f32_f32_subgroup_no_shmem_data’?
/home/xuegao/app/ai/llama.cpp/build/ggml/src/ggml-vulkan/ggml-vulkan-shaders.cpp:2956813: 错误：‘mul_mat_vec_q5_0_q8_1_f32_len’ was not declared in this scope; did you mean ‘mul_mat_vec_q5_0_f32_f32_len’?
2956813 | uint64_t arr_dmmv_q5_0_q8_1_f32_len[3] =  {mul_mat_vec_q5_0_q8_1_f32_len,  mul_mat_vec_q5_0_q8_1_f32_subgroup_len, mul_mat_vec_q5_0_q8_1_f32_subgroup_no_shmem_len};
/home/xuegao/app/ai/llama.cpp/build/ggml/src/ggml-vulkan/ggml-vulkan-shaders.cpp:2956813: 错误：‘mul_mat_vec_q5_0_q8_1_f32_subgroup_len’ was not declared in this scope; did you mean ‘mul_mat_vec_q5_0_f32_f32_subgroup_len’?
/home/xuegao/app/ai/llama.cpp/build/ggml/src/ggml-vulkan/ggml-vulkan-shaders.cpp:2956813: 错误：‘mul_mat_vec_q5_0_q8_1_f32_subgroup_no_shmem_len’ was not declared in this scope; did you mean ‘mul_mat_vec_q5_0_f32_f32_subgroup_no_shmem_len’?
/home/xuegao/app/ai/llama.cpp/build/ggml/src/ggml-vulkan/ggml-vulkan-shaders.cpp:2956814: 错误：‘mul_mat_vec_q5_1_q8_1_f32_data’ was not declared in this scope; did you mean ‘mul_mat_vec_q5_1_f32_f32_data’?
2956814 | unsigned char *arr_dmmv_q5_1_q8_1_f32_data[3] = {mul_mat_vec_q5_1_q8_1_f32_data, mul_mat_vec_q5_1_q8_1_f32_subgroup_data, mul_mat_vec_q5_1_q8_1_f32_subgroup_no_shmem_data};
/home/xuegao/app/ai/llama.cpp/build/ggml/src/ggml-vulkan/ggml-vulkan-shaders.cpp:2956814: 错误：‘mul_mat_vec_q5_1_q8_1_f32_subgroup_data’ was not declared in this scope; did you mean ‘mul_mat_vec_q5_1_f32_f32_subgroup_data’?
/home/xuegao/app/ai/llama.cpp/build/ggml/src/ggml-vulkan/ggml-vulkan-shaders.cpp:2956814: 错误：‘mul_mat_vec_q5_1_q8_1_f32_subgroup_no_shmem_data’ was not declared in this scope; did you mean ‘mul_mat_vec_q5_1_f32_f32_subgroup_no_shmem_data’?
/home/xuegao/app/ai/llama.cpp/build/ggml/src/ggml-vulkan/ggml-vulkan-shaders.cpp:2956815: 错误：‘mul_mat_vec_q5_1_q8_1_f32_len’ was not declared in this scope; did you mean ‘mul_mat_vec_q5_1_f32_f32_len’?
2956815 | uint64_t arr_dmmv_q5_1_q8_1_f32_len[3] =  {mul_mat_vec_q5_1_q8_1_f32_len,  mul_mat_vec_q5_1_q8_1_f32_subgroup_len, mul_mat_vec_q5_1_q8_1_f32_subgroup_no_shmem_len};
/home/xuegao/app/ai/llama.cpp/build/ggml/src/ggml-vulkan/ggml-vulkan-shaders.cpp:2956815: 错误：‘mul_mat_vec_q5_1_q8_1_f32_subgroup_len’ was not declared in this scope; did you mean ‘mul_mat_vec_q5_1_f32_f32_subgroup_len’?
/home/xuegao/app/ai/llama.cpp/build/ggml/src/ggml-vulkan/ggml-vulkan-shaders.cpp:2956815: 错误：‘mul_mat_vec_q5_1_q8_1_f32_subgroup_no_shmem_len’ was not declared in this scope; did you mean ‘mul_mat_vec_q5_1_f32_f32_subgroup_no_shmem_len’?
/home/xuegao/app/ai/llama.cpp/build/ggml/src/ggml-vulkan/ggml-vulkan-shaders.cpp:2956816: 错误：‘mul_mat_vec_q8_0_q8_1_f32_data’ was not declared in this scope; did you mean ‘mul_mat_vec_q8_0_f32_f32_data’?
2956816 | unsigned char *arr_dmmv_q8_0_q8_1_f32_data[3] = {mul_mat_vec_q8_0_q8_1_f32_data, mul_mat_vec_q8_0_q8_1_f32_subgroup_data, mul_mat_vec_q8_0_q8_1_f32_subgroup_no_shmem_data};
/home/xuegao/app/ai/llama.cpp/build/ggml/src/ggml-vulkan/ggml-vulkan-shaders.cpp:2956816: 错误：‘mul_mat_vec_q8_0_q8_1_f32_subgroup_data’ was not declared in this scope; did you mean ‘mul_mat_vec_q8_0_f32_f32_subgroup_data’?
/home/xuegao/app/ai/llama.cpp/build/ggml/src/ggml-vulkan/ggml-vulkan-shaders.cpp:2956816: 错误：‘mul_mat_vec_q8_0_q8_1_f32_subgroup_no_shmem_data’ was not declared in this scope; did you mean ‘mul_mat_vec_q8_0_f32_f32_subgroup_no_shmem_data’?
/home/xuegao/app/ai/llama.cpp/build/ggml/src/ggml-vulkan/ggml-vulkan-shaders.cpp:2956817: 错误：‘mul_mat_vec_q8_0_q8_1_f32_len’ was not declared in this scope; did you mean ‘mul_mat_vec_q8_0_f32_f32_len’?
2956817 | uint64_t arr_dmmv_q8_0_q8_1_f32_len[3] =  {mul_mat_vec_q8_0_q8_1_f32_len,  mul_mat_vec_q8_0_q8_1_f32_subgroup_len, mul_mat_vec_q8_0_q8_1_f32_subgroup_no_shmem_len};
/home/xuegao/app/ai/llama.cpp/build/ggml/src/ggml-vulkan/ggml-vulkan-shaders.cpp:2956817: 错误：‘mul_mat_vec_q8_0_q8_1_f32_subgroup_len’ was not declared in this scope; did you mean ‘mul_mat_vec_q8_0_f32_f32_subgroup_len’?
/home/xuegao/app/ai/llama.cpp/build/ggml/src/ggml-vulkan/ggml-vulkan-shaders.cpp:2956817: 错误：‘mul_mat_vec_q8_0_q8_1_f32_subgroup_no_shmem_len’ was not declared in this scope; did you mean ‘mul_mat_vec_q8_0_f32_f32_subgroup_no_shmem_len’?
[ 37%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/template-instances/fattn-vec-f16-instance-hs128-q5_0-q5_0.cu.o
gmake[2]: *** [ggml/src/ggml-vulkan/CMakeFiles/ggml-vulkan.dir/build.make:230：ggml/src/ggml-vulkan/CMakeFiles/ggml-vulkan.dir/ggml-vulkan-shaders.cpp.o] 错误 1
gmake[1]: *** [CMakeFiles/Makefile2:1910：ggml/src/ggml-vulkan/CMakeFiles/ggml-vulkan.dir/all] 错误 2
gmake[1]: *** 正在等待未完成的任务....
[ 37%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/template-instances/fattn-vec-f16-instance-hs128-q5_0-q5_1.cu.o
[ 37%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/template-instances/fattn-vec-f16-instance-hs128-q5_0-q8_0.cu.o
[ 38%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/template-instances/fattn-vec-f16-instance-hs128-q5_1-f16.cu.o
[ 38%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/template-instances/fattn-vec-f16-instance-hs128-q5_1-q4_0.cu.o
[ 38%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/template-instances/fattn-vec-f16-instance-hs128-q5_1-q4_1.cu.o
[ 38%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/template-instances/fattn-vec-f16-instance-hs128-q5_1-q5_0.cu.o
[ 39%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/template-instances/fattn-vec-f16-instance-hs128-q5_1-q5_1.cu.o
[ 39%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/template-instances/fattn-vec-f16-instance-hs128-q5_1-q8_0.cu.o
[ 39%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/template-instances/fattn-vec-f16-instance-hs128-q8_0-f16.cu.o
[ 39%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/template-instances/fattn-vec-f16-instance-hs128-q8_0-q4_0.cu.o
[ 39%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/template-instances/fattn-vec-f16-instance-hs128-q8_0-q4_1.cu.o
[ 40%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/template-instances/fattn-vec-f16-instance-hs128-q8_0-q5_0.cu.o
[ 40%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/template-instances/fattn-vec-f16-instance-hs128-q8_0-q5_1.cu.o
[ 40%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/template-instances/fattn-vec-f16-instance-hs128-q8_0-q8_0.cu.o
[ 40%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/template-instances/fattn-vec-f16-instance-hs256-f16-f16.cu.o
[ 41%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/template-instances/fattn-vec-f16-instance-hs64-f16-f16.cu.o
[ 41%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/template-instances/fattn-vec-f16-instance-hs64-f16-q4_0.cu.o
[ 41%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/template-instances/fattn-vec-f16-instance-hs64-f16-q4_1.cu.o
[ 41%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/template-instances/fattn-vec-f16-instance-hs64-f16-q5_0.cu.o
[ 42%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/template-instances/fattn-vec-f16-instance-hs64-f16-q5_1.cu.o
[ 42%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/template-instances/fattn-vec-f16-instance-hs64-f16-q8_0.cu.o
[ 42%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/template-instances/fattn-vec-f32-instance-hs128-f16-f16.cu.o
[ 42%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/template-instances/fattn-vec-f32-instance-hs128-f16-q4_0.cu.o
[ 42%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/template-instances/fattn-vec-f32-instance-hs128-f16-q4_1.cu.o
[ 43%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/template-instances/fattn-vec-f32-instance-hs128-f16-q5_0.cu.o
[ 43%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/template-instances/fattn-vec-f32-instance-hs128-f16-q5_1.cu.o
[ 43%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/template-instances/fattn-vec-f32-instance-hs128-f16-q8_0.cu.o
[ 43%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/template-instances/fattn-vec-f32-instance-hs128-q4_0-f16.cu.o
[ 44%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/template-instances/fattn-vec-f32-instance-hs128-q4_0-q4_0.cu.o
[ 44%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/template-instances/fattn-vec-f32-instance-hs128-q4_0-q4_1.cu.o
[ 44%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/template-instances/fattn-vec-f32-instance-hs128-q4_0-q5_0.cu.o
[ 44%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/template-instances/fattn-vec-f32-instance-hs128-q4_0-q5_1.cu.o
[ 44%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/template-instances/fattn-vec-f32-instance-hs128-q4_0-q8_0.cu.o
[ 45%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/template-instances/fattn-vec-f32-instance-hs128-q4_1-f16.cu.o
[ 45%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/template-instances/fattn-vec-f32-instance-hs128-q4_1-q4_0.cu.o
[ 45%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/template-instances/fattn-vec-f32-instance-hs128-q4_1-q4_1.cu.o
[ 45%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/template-instances/fattn-vec-f32-instance-hs128-q4_1-q5_0.cu.o
[ 46%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/template-instances/fattn-vec-f32-instance-hs128-q4_1-q5_1.cu.o
[ 46%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/template-instances/fattn-vec-f32-instance-hs128-q4_1-q8_0.cu.o
[ 46%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/template-instances/fattn-vec-f32-instance-hs128-q5_0-f16.cu.o
[ 46%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/template-instances/fattn-vec-f32-instance-hs128-q5_0-q4_0.cu.o
[ 47%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/template-instances/fattn-vec-f32-instance-hs128-q5_0-q4_1.cu.o
[ 47%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/template-instances/fattn-vec-f32-instance-hs128-q5_0-q5_0.cu.o
[ 47%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/template-instances/fattn-vec-f32-instance-hs128-q5_0-q5_1.cu.o
[ 47%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/template-instances/fattn-vec-f32-instance-hs128-q5_0-q8_0.cu.o
[ 47%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/template-instances/fattn-vec-f32-instance-hs128-q5_1-f16.cu.o
[ 48%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/template-instances/fattn-vec-f32-instance-hs128-q5_1-q4_0.cu.o
[ 48%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/template-instances/fattn-vec-f32-instance-hs128-q5_1-q4_1.cu.o
[ 48%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/template-instances/fattn-vec-f32-instance-hs128-q5_1-q5_0.cu.o
[ 48%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/template-instances/fattn-vec-f32-instance-hs128-q5_1-q5_1.cu.o
[ 49%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/template-instances/fattn-vec-f32-instance-hs128-q5_1-q8_0.cu.o
[ 49%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/template-instances/fattn-vec-f32-instance-hs128-q8_0-f16.cu.o
[ 49%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/template-instances/fattn-vec-f32-instance-hs128-q8_0-q4_0.cu.o
[ 49%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/template-instances/fattn-vec-f32-instance-hs128-q8_0-q4_1.cu.o
[ 49%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/template-instances/fattn-vec-f32-instance-hs128-q8_0-q5_0.cu.o
[ 50%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/template-instances/fattn-vec-f32-instance-hs128-q8_0-q5_1.cu.o
[ 50%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/template-instances/fattn-vec-f32-instance-hs128-q8_0-q8_0.cu.o
[ 50%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/template-instances/fattn-vec-f32-instance-hs256-f16-f16.cu.o
[ 50%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/template-instances/fattn-vec-f32-instance-hs64-f16-f16.cu.o
[ 51%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/template-instances/fattn-vec-f32-instance-hs64-f16-q4_0.cu.o
[ 51%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/template-instances/fattn-vec-f32-instance-hs64-f16-q4_1.cu.o
[ 51%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/template-instances/fattn-vec-f32-instance-hs64-f16-q5_0.cu.o
[ 51%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/template-instances/fattn-vec-f32-instance-hs64-f16-q5_1.cu.o
[ 52%] Building HIP object ggml/src/ggml-hip/CMakeFiles/ggml-hip.dir/__/ggml-cuda/template-instances/fattn-vec-f32-instance-hs64-f16-q8_0.cu.o
[ 52%] Linking HIP shared library ../../../bin/libggml-hip.so
clang++: warning: argument unused during compilation: '--gpu-max-threads-per-block=1024' [-Wunused-command-line-argument]
[ 52%] Built target ggml-hip
gmake: *** [Makefile:146：all] 错误 2
```

same issue here ( using MSVC 2022 on Windows as an example )

ggml-vulkan-shaders.cpp
ggml\src\ggml-vulkan\ggml-vulkan-shaders.cpp(2951735,50): error C2065: 'mul_mat_vec_q4_0_q8_1_f32_data': undeclared identifier
ggml\src\ggml-vulkan\ggml-vulkan-shaders.cpp(2951735,82): error C2065: 'mul_mat_vec_q4_0_q8_1_f32_subgroup_data': undeclared identifier

etc ...

--- a/ggml/src/ggml-vulkan/vulkan-shaders/vulkan-shaders-gen.cpp
+++ b/ggml/src/ggml-vulkan/vulkan-shaders/vulkan-shaders-gen.cpp
@@ -854,7 +854,13 @@ void write_output_files() {
         fputs(len.c_str(), src);
     }
 
-    for (const std::string& btype : {"f16", "f32", "q8_1"}) {
+    std::vector<std::string> btypes = {"f16", "f32"};
+
+#if defined(GGML_VULKAN_INTEGER_DOT_GLSLC_SUPPORT)
+    btypes.push_back("q8_1");
+#endif
+
+    for (const std::string& btype : btypes) {
     for (const auto& tname : type_names) {
         if (btype == "q8_1" && !is_legacy_quant(tname)) {
             continue;

